### PR TITLE
EN: EP3 Dialogue and Minor Omnibus EP1 (Nearly done)

### PR DESCRIPTION
**Pull request by SorryLorry**
Please go ahead and make any changes if necessary. Definitely needs some grammar check.

### CODE REVIEW

File: Story/Episode 3/np_010851.csv [hunk 1]
```diff
@@ -0,0 +1,53 @@
+11_voice_NPC_5085_104#0,"""About ARKS... Let's start from the Beginning of Arks."""
+text01002#0,"""The name \""ARKS\"" was given to the team that investigates the various planets in the Universe."""
+text01003#0,"""The purpose of ARKS is not simply conducting investigations. They are also responsible for annihilating Darkers or any enemies corrupted by Darkers."""
```
Comment (DabirSA) [inline Story/Episode 3/np_010851.csv]: "Darkers, or any enemies corrupted by Darkers."

File: Story/Episode 3/np_010851.csv [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+text01004#0,"""ARKS was first created by a Civilization known as the Photoners."""
```
Comment (DabirSA) [inline Story/Episode 3/np_010851.csv]: "a civilization known as the Photoners."

File: Story/Episode 3/np_010851.csv [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+text01005#0,"""No... ARKS was created by a Being revered as a God by Photoners known as \""Xion\"". """
```
Comment (DabirSA) [inline Story/Episode 3/np_010851.csv]: "a being revered as a God by Photoners, known as \\""Xion\\""."

File: Story/Episode 3/np_010851.csv [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+text01006#0,"""All for the purpose of defeating Darkers, and act as the Guardians of the Universe"""
```
Comment (DabirSA) [inline Story/Episode 3/np_010851.csv]: "and to act as the guardians of the universe,"

File: Story/Episode 3/np_010851.csv [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+text01007#0,"""Over the years, the objectives of ARKS seemed to have veered off course. Somehow, I managed to guide it back. """
+11_voice_NPC_5085_100#0,"""Exploring the Universe, Investigate Planets. Encountering any Darkers, fight and defeat them. That is ARKS."""
```
Comment (DabirSA) [inline Story/Episode 3/np_010851.csv]: "Explore the universe, investigate planets. When encountering Darkers, fight and defeat them."

File: Story/Episode 3/np_010851.csv [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+11_voice_NPC_5085_106#0,"""About the Oracle huh? Oracle is basically an enormous fleet of Ships."""
```
Comment (DabirSA) [inline Story/Episode 3/np_010851.csv]: "About Oracle, huh?"

File: Story/Episode 3/np_010851.csv [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+text02002#0,"""Each ARKS Ship serves as a base of operations for ARKS and also as the residence for its citizens."""
+text02003#0,"""The fleet is made up of such ships."""
+text02004#0,"""Currently, we are in just one of the many Ships. Just a section of this Ship. """
+text02005#0,"""And the ship at the heart of the fleet, the \""Mothership\"", houses the core of my existence."""
+11_voice_NPC_5085_108#0,"""Previously, Xion was the core of the Mothership. However, we lost her to [Loser]. """
+text02007#0,"""Since then, I have succeeded Xion, assuming control of the whole fleet."""
+text02008#0,"""That sounds confusing huh.. To put it simply, Oracle is under my control."""
```
Comment (DabirSA) [inline Story/Episode 3/np_010851.csv]: "That sounds confusing, huh." Watch your number of full stops.

File: Story/Episode 3/np_010851.csv [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+text02009#0,"""Though it might get lonely without Xion... I have to push on."""
+11_voice_NPC_5085_100#1,"""For the future of Oracle. For ARKS. For you and me."""
+text03001#0,"""About Darkers... They spread throughout the Universe. An enemy of the Universe, an enemy of every living being."""
```
Comment (DabirSA) [inline Story/Episode 3/np_010851.csv]: "the universe. An enemy of the universe, an enemy of all life."

File: Story/Episode 3/np_010851.csv [hunk 9]
```diff
@@ -0,0 +1,53 @@
...
+text03002#0,"""Darkers consume all in their path. Every civilization, every living thing."""
+text03003#0,"""Only ARKS can permanently defeat Darkers with the use of Photons."""
+text03004#0,"""Even if someone is able to defeat the Darkers without the use of Photons, the residual fragments will eventually corrupt him. """
```
Comment (DabirSA) [inline Story/Episode 3/np_010851.csv]: "eventually corrupt them." Watch that trailing space.

File: Story/Episode 3/np_010851.csv [hunk 10]
```diff
@@ -0,0 +1,53 @@
...
+11_voice_NPC_5085_105#0,"""Under the Darkers corruption, they lose any consciousness of themselves, mutating into a ferocious enemy."""
+text03006#0,"""This is not only limited to organic beings. Mechanical beings and Artificial Intelligence are equally susceptible. """
+text03007#0,"""There are variances in mutation, however they end up corrupted all the same. Therefore, only ARKS is able to defeat Darkers."""
+11_voice_NPC_5085_104#1,"""Other means of defeating them may exist, if you consider the endless possibilities of the Universe."""
+text03009#0,"""For now, only ARKS is able to exterminate the Darkers. Which is why we are travelling across the Universe."""
+11_voice_NPC_5085_104#2,"""Dark Falz... To be honest, I do not have much information regarding them... So this will be short."""
+text04002#0,"""Currently, there is 5 confirmed existence of Dark Falz. [Elder], [Apprentice], [Loser], [Double] and lastly, [Persona]"""
```
Comment (DabirSA) [inline Story/Episode 3/np_010851.csv]: "Currently, there are 5 confirmed Dark Falz."

File: Story/Episode 3/np_010851.csv [hunk 11]
```diff
@@ -0,0 +1,53 @@
...
+text04003#0,"""I can't say much about their characteristics... Well, you can simply think of them as the Commanders of Darkers."""
+text04004#0,"""[Elder] and [Apprentice] is more active than others in attacking ARKS."""
```
Comment (DabirSA) [inline Story/Episode 3/np_010851.csv]: "[Elder] and [Apprentice] are"

File: Story/Episode 3/np_010851.csv [hunk 12]
```diff
@@ -0,0 +1,53 @@
...
+text04005#0,"""[Loser] was defeated by Regius.. Due to [Double]'s power however, clones of [Loser] appear frequently."""
```
Comment (DabirSA) [inline Story/Episode 3/np_010851.csv]: "[Loser] was defeated by Regius, but was devoured by [Double], and so clones of it appear frequently."

File: Story/Episode 3/np_010851.csv [hunk 13]
```diff
@@ -0,0 +1,53 @@
...
+text04006#0,"""As for [Persona]... Honestly, I dont know much."""
+text04007#0,"""Unless we are able to gather more information, there's nothing I can share. Sorry."""
+text04008#0,"""One thing is for sure, we will not have peace until Dark Falz is defeated."""
```
Comment (DabirSA) [inline Story/Episode 3/np_010851.csv]: "until the Dark Falz are defeated."

File: Story/Episode 3/np_010851.csv [hunk 14]
```diff
@@ -0,0 +1,53 @@
...
+11_voice_NPC_5085_100#2,"""It's tough. That's why ARKS has to do it, only ARKS can do it."""
+11_voice_NPC_5085_100#3,"""Matter Board... I can tell you what I understand about it."""
```
Comment (DabirSA) [inline Story/Episode 3/np_010851.csv]: "The Matter Board..."

File: Story/Episode 3/np_010851.csv [hunk 15]
```diff
@@ -0,0 +1,53 @@
...
+text05002#0,"""As Xion was capable of calculating every possiblity in this Universe. She was able to calculate her desired outcome."""
```
Comment (DabirSA) [inline Story/Episode 3/np_010851.csv]: "in this universe, she""

File: Story/Episode 3/np_010851.csv [hunk 16]
```diff
@@ -0,0 +1,53 @@
...
+text05003#0,"""<%me>.<br>Matter Board was created for you to change the future."""
```
Comment (DabirSA) [inline Story/Episode 3/np_010851.csv]: "The Matter Board was created to allow you"

File: Story/Episode 3/np_010851.csv [hunk 17]
```diff
@@ -0,0 +1,53 @@
...
+text05004#0,"""Besides making it possible to change the future, you could travel back in time to correct the past."""
+text05005#0,"""Matter Board allowed you to save people that you could not save before, or chancing upon encounters that were not supposed to happen"""
```
Comment (DabirSA) [inline Story/Episode 3/np_010851.csv]: "The Matter Board"
"or chance upon encounters that were not supposed to happen."

File: Story/Episode 3/np_010851.csv [hunk 18]
```diff
@@ -0,0 +1,53 @@
...
+11_voice_NPC_5085_108#1,"""However, with Xion gone. I am unable to create a perfect Matter Board"""
```
Comment (DabirSA) [inline Story/Episode 3/np_010851.csv]: "with Xion gone, I am"
"a perfect Matter Board."

File: Story/Episode 3/np_010851.csv [hunk 19]
```diff
@@ -0,0 +1,53 @@
...
+text05007#0,"""My Matter Boards are simply an imitation of Xion's, weaker than Xion's."""
```
Comment (DabirSA) [inline Story/Episode 3/np_010851.csv]: "are simply a weaker imitation of Xion's."

File: Story/Episode 3/np_010851.csv [hunk 20]
```diff
@@ -0,0 +1,53 @@
...
+text05008#0,"""Although it is still capable of time travel and changing the future to a certain degree... Making significant changes to the past is simply out of the question."""
```
Comment (DabirSA) [inline Story/Episode 3/np_010851.csv]: "Although they are still capable"

File: Story/Episode 3/np_010851.csv [hunk 21]
```diff
@@ -0,0 +1,53 @@
...
+text05009#0,"""Because the Matter Board is the product of Xion's wishes, I am unable to fully grasp it."""
+text05010#0,"""However, I am able to understand and use Matter Boards left behind by Xion."""
+11_voice_NPC_5085_100#4,"""For now... Only I am able understand it. I should arrange it for people to understand it easier huh?"""
```
Comment (DabirSA) [inline Story/Episode 3/np_010851.csv]: "am able to understand it."
"understand it more easily, huh?"

**Review by DabirSA — COMMENTED** [review]
Grammar and punctuation, and a couple wording change suggestions that are probably down to taste. You have a lot of odd capitalisation on common nouns.

File: Story/Episode 3/un_034090.csv [hunk 22]
```diff
@@ -0,0 +1,26 @@
+name01#0,"""？？？"""
+name02#0,"""？？？"""
+un_034090_0010#0,"""Ah, it's you.<br>Could you entertain Rela for me?"""
+un_034090_0020#0,"""Light is running away again. He gets frightened everytime Rela greets him with a roar. I'll drag him back here."""
```
Comment (alama) [inline Story/Episode 3/un_034090.csv]: `everytime` or `every time`?

File: Story/Episode 3/np_010852.csv [hunk 23]
```diff
@@ -0,0 +1,150 @@
+11_voice_NPC_5085_104#0,"""The current situation?<br>It's a long story so I'll just summarize it for you."""
+text01002#0,"""First,changes in organization.<br>Void, the organization that did<br>inhumane studies started 40 years ago."""
+text01003#0,"""Luther was the leader.<br>He took the knowledge and power<br>from Xion and controlled both<br>Oracle and ARKS behind the scenes."""
+11_voice_NPC_5085_103#0,"""Luther said it was to understand Xion, but he just wanted her power."""
+text01005#0,"""However, half a year ago<br>With <%me>'s cooperation,<br>we managed to wrest control from Luther."""
+text01006#0,"""We discovered that Luther was<br>Dark Falz【Loser】. ARKS managed<br>to defeat him."""
+text01007#0,"""Oracle was released from the<br>Photonners... No, from Luther's<br>control."""
```
Comment (alama) [inline Story/Episode 3/np_010852.csv]: Why "Photonners"?

File: Story/Episode 3/np_010852.csv [hunk 24]
```diff
@@ -0,0 +1,150 @@
...
+11_voice_NPC_5085_104#2,"""Dark Falz【Persona】.<br>Truth be told, he remains a mystery to us."""
+text03002#0,"""He does not control specific<br>Darker types.<br>However, he constantly interferes<br>with <%me> and Matoi."""
+text03003#0,"""As we have seen, 10 years ago,<br>【Persona】appeared in front 2nd Generation Claris Claes;Matoi.<br>Just to deceive her."""
+text03004#0,"""From appearance to perimeters,<br>every detail was an exact match<br>with you. He too, was able to<br>travel across time."""
+text03005#0,"""How was【Persona】capable of such feat is a mystery."""
+text03006#0,"""It would be best to assume that<br>【Persona】is equally unique as you,<br>possessing exceptional qualities."""
+text03007#0,"""However, I do have a hypothesis<br>but that requires further<br>investigations. I'll contact you<br>if there are further findings."""
+11_voice_NPC_5085_104#3,"""Analyzation of the data collected<br>from Wopal's Facility is<br>progressing smoothly. The data<br>has allowed us to know more<br>about Wopal."""
+text04002#0,"""Originally, Planet Wopal was a<br>designed as a resort for<br>Photonners."""
```
Comment (alama) [inline Story/Episode 3/np_010852.csv]: Photonners?

File: Story/Episode 3/np_010852.csv [hunk 25]
```diff
@@ -0,0 +1,150 @@
...
+text04003#0,"""The ruins of an amusement park<br>and shiny structures can be<br>found."""
+text04004#0,"""However, something regarding<br>【Profound Darkness】caused the<br>resort to be abandoned."""
+text04005#0,"""As the Photonners travelled<br>through the universe, Luther<br>noticed the facility."""
+text04006#0,"""Not only was the facility<br>remodelled to do experiments,<br>it seems as if the planet was<br>remodelled as well."""
```
Comment (alama) [inline Story/Episode 3/np_010852.csv]: Photonners?

File: Story/Episode 3/np_010852.csv [hunk 26]
```diff
@@ -0,0 +1,150 @@
...
+11_voice_NPC_5085_100#1,"""It was surprising that she<br>believed a message that came<br>from an unknown sender."""
+text05008#0,"""But I digress. Let's get back<br>to the topic."""
+text05009#0,"""Luther's experiments released<br>the Photons within Sara's body,<br>thus disabling her ability to<br>ever use Techniques again."""
+text05010#0,"""In short. The photons in her<br>body now is supplied by me."""
+text05011#0,"""Thus, Sara is connected to me."""
+text05012#0,"""Moving on, Claris Claes is a<br>clone of Sara, using Sara's data."""
+text05013#0,"""She was created to use an<br>imitation of Clarissa, which<br>pushed the limits of Techniques."""
+text05014#0,"""Her body age was set to be the<br>same as Sara, with certain<br>growth forced upon it. Her actual<br>age is actually 5-6 years old."""
+text05015#0,"""Well. Nothing can be done at<br>this point."""
+text05016#0,"""Both of them decided to live<br>their lives seperately as Sara<br>and Claris Claes."""
```
Comment (alama) [inline Story/Episode 3/np_010852.csv]: seperately or separately?

**Review by alama — COMMENTED** [review]
You got some spelling mistakes

File: Story/Episode 3/np_010852.csv [hunk 27]
```diff
@@ -65,16 +65,16 @@ text06007#0,"""Besides, there is a possibility<br>that this may occur to other<b
...
 text07004#0,"""I do not wish to return to the<br>state where my consciousness was<br>in that pool of water. I want<br>to be like everyone else here."""
 text07005#0,"""Enough with the small talk...<br>Sukuna-hime is the god of good,<br>while Magatsu is a god of evil<br>and annihilation, the god that<br>harms the planet."""
 text07006#0,"""Be it Shironians or Kuronites,<br>it's destruction knows no bounds."""
 text07007#0,"""Sukuna-hime has sealed Magatsu<br>a long time ago but it seems<br>that the power of the seals are<br>fading."""
 text07008#0,"""While reapplying the seal,<br>【Double】's interference caused<br>a part of Magatsu to escape."""
-text07009#0,"""I am more concerned about how<br>【Double】is going to use Magatsu's<br>power than simply the fact that<br>a part of Magatsu escaped."""
-text07010#0,"""This isnt simply a problem for<br>Harkotan, but the universe as<br>well."""
+text07009#0,"""I am more concerned about how<br>【Double】is going to use Magatsus<br>power than simply the fact that<br>a part of Magatsu escaped."""
```
Comment (alama) [inline Story/Episode 3/np_010852.csv]: Should this be "Magatsus's power", not "Magatsus Power"?

Comment (SynthSy) [reply Story/Episode 3/np_010852.csv]: Magatsu's and 【】need to be replaced with [ and ]

File: Story/Episode 3/un_034060.csv [hunk 28]
```diff
@@ -0,0 +1,23 @@
...
+un_034060_0090#0,"""I.. I understand."""
+un_034060_0100#0,"""Then get to it, I am not exactly<br>free to nag at you as well."""
+un_034060_0110#0,"""Oh, if it isn't <%me>.<br>Do you need anything?"""
+un_034060_0120#0,"""1. It must be difficult huh."""
+un_034060_0130#0,"""2. Aren't you being too strict?"""
+un_034060_0140#0,"""Thanks for your concern,<br>however it is my job after all."""
+un_034060_0150#0,"""Not at all. All of this is part<br>of my job, I do not mix personal<br>matters with official work."""
+un_034060_0160#0,"""By pointing out small errors,<br>they will be more meticulous.<br>This reduces the mistakes they<br>make in the long run."""
+un_034060_0170#0,"""ARKS is going through a shift in<br>structure. Which is why setting<br>things right is important."""
+un_034060_0180#0,"""Because of Luther, Regius is<br>still cast under a negative light.<br>The other members of the<br>Council of Six is out of the<br>question."""
```
Comment (SynthSy) [inline Story/Episode 3/un_034060.csv]: Remove the line breaks here please.

File: Story/Episode 3/un_034080.csv [hunk 29]
```diff
@@ -0,0 +1,24 @@
+name01#0,"""？？？"""
+name02#0,"""？？？"""
+un_034080_0010#0,"""Ah,<%me>.<br>What a coincidence."""
+un_034080_0020#0,"""I wanted to interact with the Lillipans, thus I asked Fourier for help but...."""
+un_034080_0030#0,"""Lilili!\u3000Lilili!"""
```
Comment (SynthSy) [inline Story/Episode 3/un_034080.csv]: The `\u3000` can be replaced with a space.

File: Story/Omnibus/st_010230_om.csv [hunk 30]
```diff
@@ -4,12 +4,12 @@ st_010230_0010#0,"""In all things, all possible outcomes cannot come to pass. On
...
-al_010030_0010#0,"""ダーカーって、ほんっとやっかいだよね。<br>本体を消滅させても残ったダーカー因子が<br>生物や兵器を侵食して暴走させるとかさ。"""
-al_010030_0020#0,"""よっくもまあ、そんなやっかいなのを<br>作り上げたものだと、正直に思うよ。"""
-al_010030_0030#0,"""わたしたちだって、体内のフォトンが<br>ダーカー因子を中和してくれなかったら<br>あっという間に侵食されちゃうんだ。"""
-al_010030_0040#0,"""だから、アークス以外は戦えない。<br>フォトンを扱う才能がなければ<br>ダーカーと戦うことは、できない。"""
-al_010030_0050#0,"""わたしやギーくん、リーちゃんは<br>その才能がすごかったって話だけど<br>貴方はその比じゃない。"""
-al_010030_0060#0,"""凶暴化しかかった龍族の内にある<br>ダーカー因子を吸収して浄化とか<br>……わたしには無理かな。"""
-al_010030_0070#0,"""正直、怖いぐらいの能力だよ。<br>でもその力も、シオンが貴方を選んだ<br>理由の一つなんじゃないかな。"""
-al_010030_0080#0,"""おかげで、凶暴化するはずだった龍族が<br>正気に戻るという未来に書き換わり<br>アークスと龍族の間に、交流が生まれた。"""
-al_010030_0090#0,"""これが未来にどんな変化をもたらすか<br>まだわからないけど、シオンの目的には<br>必要不可欠な道筋、だったんだろうね。"""
+al_010030_0010#0,"""Darkers are a real pain aren't they.<br>Even after defeating them,<br>residual fragments corrupt organic<br>and inorganic alike,<br>causing them to go berserk."""
```
Comment (SynthSy) [inline Story/Omnibus/st_010230_om.csv]: Don't preserve line breaks

File: Story/Episode 3/un_034040.csv [hunk 31]
```diff
@@ -0,0 +1,29 @@
+name01#0,"""？？？"""
+name02#0,"""？？？"""
+un_034040_0010#0,"""Yo <%me>!<br>You seem to be doing fine in ARKS."""
+un_034040_0020#0,"""I heard a lot of good things<br>about you."""
+un_034040_0030#0,"""I know that you are capable,<br>but you should take a break too."""
```
Comment (SynthSy) [inline Story/Episode 3/un_034040.csv]: Again, don't preserve linebreaks.

File: Story/Episode 3/ra_034060.csv [hunk 32]
```diff
@@ -3,12 +3,12 @@ name02#0,"""???"""
 ra_034060_0001#0,"""A.P.239/3/1"""
 ra_034060_0010#0,"""I see...<br>The Dark Falz that appeared ten years ago;<br>[Took form of a child]."""
 ra_034060_0020#0,"""If one person confirmed that,<br>there's a possibility that their<br>memory's wrong.<br>But, if other people says the same,<br>then we're not mistaken."""
-ra_034060_0030#0,"""It seems like [Apprentice] wasn't the only Dark Falz<br>that appeared ten years ago, """
+ra_034060_0030#0,"""It seems like [Apprentice] wasn't the only Dark Falz that appeared<br>ten years ago, """
```
Comment (SynthSy) [inline Story/Episode 3/ra_034060.csv]: Don't preserve linebreaks here.

File: Story/Episode 3/np_010852.csv [hunk 33]
```diff
@@ -0,0 +1,150 @@
+11_voice_NPC_5085_104#0,"""The current situation?<br>It's a long story so I'll just<br>summarize it for you."""
```
Comment (SynthSy) [inline Story/Episode 3/np_010852.csv]: This entire document has punctuation mistakes.

Also, do not preserve the linebreaks.

**Review by SynthSy — CHANGES_REQUESTED** [review]
Overall, this PR needs a major re-work of the dialog and punctuation. Too many errors relating to sentence structure, as well as terms, and a complete disregard to formatting. These need to be corrected before this can be merged.

File: Story/Episode 3/np_010852.csv [hunk 34]
```diff
@@ -1,92 +1,92 @@
...
+text01008#0,"""After the death of Xion, I took over her in managing the fleet."""
+text01009#0,"""A lot of things happened since but I think that we are in a good spot for now."""
+text01010#0,"""After all, Xion did her best for this outcome..."""
+11_voice_NPC_5085_104#1,"""Matoi... She's been through difficult times."""
+text02002#0,"""Previously, we travelled 10 years back to know more about Matoi's past and the reason behind her deleted records."""
+text02003#0,"""Matoi... 2nd generation Claris Claes in a battle against Darkers 10 years ago, nearly turned into [Profound Darkness] due to the accumulated Darkness within her."""
+text02004#0,"""In order to stop herself from turning, she attempted to take her own life."""
+text02005#0,"""<%me> was present at that point, investigating the past. She was caught with you when I pulled you out of that situation."""
+text02006#0,"""You then stumbled upon her 10 years after, having lost both her memories and power."""
+text02007#0,"""If her loss was permanent, it wouldnt have made a difference. However, not only has she regained her powers, she managed to reunite with Clarissa."""
```
Comment (alama) [inline Story/Episode 3/np_010852.csv]: `wouldnt` or `wouldn't`

Comment (Kyle873) [reply Story/Episode 3/np_010852.csv]: Wouldn't.

**Comment by Aida-Enna** [comment]
Closed due to inactivity. @SorryLorry Feel free to comment on here if you're back and wish to pick up where you left off.

**Pull request closed without merging**